packages/docs/src/routes/docs/components/lite-components/index.mdx

```diff
@@ -34,3 +35,6 @@ In the above example, the `MyButton` is a lite component. Lite components' lazy
 - `MyButton` does not have a host element.
 
 You can think of lightweight components as being inlined into the component, where they are instantiated.
+
+> **Limitations**
+> Unlike regular Qwik components, lite components can not use `use` methods, such as `useSignal` or `useStore`. As it name implies, lite components are better used sparkly for lightweight pieces of markup since they offer the convenience of being bundled with the parent component.
```

Comment:
did you mean sparsely, not sparkly?

Reply:
https://github.com/BuilderIO/qwik/pull/2127

Reply:
merged, thanks @tuurbo !